src/client.ts

```diff
@@ -136,7 +136,10 @@ export class C8Client extends Fabric {
     csvPath: string,
     opts?: DocumentSaveOptions | boolean
   ) {
-    const data = await csv().fromFile(csvPath);
+    const file = fs.readFileSync(csvPath, { encoding: 'utf8', flag: 'r' });
```

Comment:
Using `fs` here will make this only work in node. jsc8 should run for both node and web JS runtimes.

Reply:
Working on it